docs/guides/run-capability.md

```diff
@@ -206,51 +215,52 @@ async function main() {
 
-You can then run your app which should perform the use case.
+For details on SuperfaceClient API, please consult [OneSDK reference](/reference/one-sdk-js).
+:::
+
+## Run the app
+
+You can then run your app that performs the use case.
 
 ```shell
 node app.js
 ```
 
-### Test the provider map
-
-Since profile & provider are simple descriptions of entities and actions, we don't usually test them. However the maps typically contain non-trivial logic that benefits from being properly tested. In addition, you might want to run integration tests against the provider's sandbox or live servers.
-
-Testing is easy since you simply perform the use case via OneSDK. Then, you test it as any other function. The only difference is that you want to make OneSDK use a specific provider when running in test. You can do that by specifying `provider` parameter inside options for `.perform` method.
-
-See the example of a test written using [`jest`](https://jestjs.io) framework:
+:::tip Offline Use
+To use capibility without the use of Superface [remote registry](https://superface.ai/catalog). You have to import capabilities into project and ensure `super.json` has valid paths to your capabilities.
 
-```javascript title="profile.provider.test.js" {10,16}
-require('dotenv').config();
-const { SuperfaceClient } = require('@superfaceai/one-sdk');
+TODO: link to offline use guide
+:::
 
-describe('scope/profile-name', () => {
-  it('should return a result when called with ...', async () => {
-    const sdk = new SuperfaceClient();
+### DEBUG
 
-    const profile = await sdk.getProfile('scope/profile-name');
+Superface uses [DEBUG package](https://github.com/visionmedia/debug), which you can use when running or testing your app. To use it, you simply have to add environment variable `DEBUG="superface*"` before running tests.
 
-    const provider = await sdk.getProvider('provider-name');
+There are mutiple parts of Superface, which implemented this package and created debug context:
 
-    const result = await profile.getUseCase('UseCaseName').perform(
-      {
-        /* Input object as defined in the profile */
-      },
-      { provider }
-    );
+in `OneSDK`:
 
-    expect(result.isOk()).toBe(true);
-  });
-});
-```
+- `superface:config`
+- `superface:profile-parameter-validator`
+- `superface:map-interpreter`
+- `superface:http`, `superface:http:sensitive`
+- `superface:superjson`
+- `superface:failover`, `superface:failover:sensitive`
+- `superface:bound-profile-provider`
+- `superface:registry`
+- `superface:usecase`
+- `superface:lib/env`
+- `superface:events`
+- `superface:metric-reporter`
 
-For unit testing all of the Map's logic, you might need to mock the specific responses. We recommend using [`nock`](https://www.npmjs.com/package/nock) for this.
+in `Parser`:
 
-## Examples
+- `superface-parser:map-validator`
+- `superface-parser:profile-io-analyzer`
```

Comment:
I think this isn't much useful without further explanation _why_ I should use these contexts. I would move it to reference/README for OneSDK and link it here.

However, I would explicitly mention `superface:http*` which is extremely useful for debugging API calls, perhaps with a note about `:sensitive` context.